<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>678</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MyWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>241</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;
font-weight : bold;</string>
    </property>
    <property name="title">
     <string>login</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;font color = &quot;white&quot;&gt;Username&lt;/font&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="login_username">
         <property name="styleSheet">
          <string notr="true">color:white;
border  : 2px solid rgba(100,100,100,0.5);
border-color : white;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;font color = &quot;white&quot;&gt;Password &lt;/font&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="login_password">
         <property name="styleSheet">
          <string notr="true">color : white;
border  :2px solid rgba(100,100,100,0.5);
border-color : white;</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_user_login">
       <property name="styleSheet">
        <string notr="true">color: White;
font-weight:bold;
border  : 2px solid rgba(100,100,100,0.5);
border-color : white;

</string>
       </property>
       <property name="text">
        <string>login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_signup">
       <property name="styleSheet">
        <string notr="true">color : white;
font-weight : bold;
border  : 2px solid rgba(100,100,100,0.5);
border-color : white;</string>
       </property>
       <property name="text">
        <string>Creating New?</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>221</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_admin_login">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #fff;
background-color: rgb(0, 0, 0);
font-weight : bold;
border  : 2px solid rgba(100,100,100,0.5);
border-color : white;

</string>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font-size : 20px;</string>
    </property>
    <property name="text">
     <string>                          Welcome to Shopno's Grocery      </string>
    </property>
   </widget>
   <widget class="QLabel" name="digital_clock">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
